molgenis-navigator/src/main/frontend/src/store/actions.js

```diff
@@ -59,34 +59,21 @@ function toEntity (item: any) {
 }
 
 /**
- * Get all ( first 1000 ) packages from the server
- * @returns {Promise}, on success resolves to list of 'all' packages, on failure resolves to error message
+ * Get a MOLGENIS rest api encoded query for the Package EntityType
```

Comment:
the fact that we 'only' get the first 1000 items seems important to me, i suggest to keep this information in the js docs